Services/Models/Simulation.cs

```diff
@@ -100,8 +100,6 @@ public Simulation()
             this.PartitioningComplete = false;
             this.CustomDevices = new List<CustomDeviceRef>();
             this.Statistics = new StatisticsRef();
-
-            this.PartitioningComplete = false;
```

Comment:
I would leave the default value explicit, doesn't do any harm

Reply:
This line was redundant with line 100.